file.go

```diff
@@ -533,7 +533,7 @@ func (f *File) Seq() int {
 // This does not align with the semantics of undo.Buffer.
 // Each "Mark" needs to have a seq value provided.
 // TODO(rjk): Consider providing the target seq value as an argument.
-func (f *File) Undo(isundo bool) (q0p, q1p int) {
+func (f *File) Undo(isundo bool) (q0p, q1p int, ok bool) {
```

Comment:
It would be good here to have some kind of explanation of what `ok` means via comment?

Reply:
Done